app/components/Network/CanvasNetworkGraph.js

```diff
@@ -112,7 +108,7 @@ class CanvasNetworkGraph extends Component {
 
     // remove active class
     d3.selectAll('.active-peer')
-      .each(function () {
```

Comment:
es6 arrow functions lexically bind `this`, so i did not use them here on purpose. may need to disable these lines for lint